composer.json

```diff
@@ -70,6 +70,12 @@
         "update-dev": [
             "@composer update",
             "@cs-setup"
+        ],
+        "update-locales": [
+            "node_modules/ttag-cli/bin/ttag extract --o src/Locale/en_US/default-js.po --l en src/Template/Layout/js/app",
```

Comment:
Problem: `it_IT` and `en_US` are not constant predefined locales but the only ones available for now. 

In `bin/cake gettext update` script the rule is to use every locale defined in `src/Locale` we should use same strategy here.